pyproject.toml

```diff
@@ -16,8 +16,20 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
     "ipykernel", # Support for Jupyter notebooks
+    "DateTimeRange>=2.0.0,<3.0.0",
```

Comment:
@IshikaKhandelwal the following packages are probably not required:

```
    "ipykernel",
     "numba>=0.57.0,<1.0.0",
    "pycwt>=0.3.0a22,<1.0.0",
    "psutil>=5.9.5,<6.0.0",
```

Reply:
there are a few files such as utils.py and noise_module.py that have to be moved to this package as the store file leverages them. psutil and numba are therefore required, will remove pycwt and ipykernel